applications/nrf5340_audio/unicast_client/README.rst

```diff
@@ -0,0 +1,146 @@
+.. _nrf53_audio_unicast_client_app:
+
+nRF5340 Audio: Unicast client
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF5340 Audio unicast client application implements the :ref:`CIS gateway mode <nrf53_audio_app_overview>`.
+
+In this mode, one Connected Isochronous Group (CIG) can be used with two Connected Isochronous Streams (CIS).
+Transmitting unidirectional or transceiving bidirectional audio happens using CIG and CIS.
+
+The following limitations apply to this application:
+
+* One CIG with two CIS.
+* Audio input: USB or I2S (Line in or using Pulse Density Modulation).
+* Audio output: USB or I2S/Analog headset output.
+* Configuration: 16 bit, several bit rates ranging from 32 kbps to 124 kbps.
+
+.. _nrf53_audio_unicast_client_app_requirements:
+
+Requirements
+************
+
+The application shares the :ref:`requirements common to all nRF5340 Audio application <nrf53_audio_app_requirements>`.
+
+.. _nrf53_audio_unicast_client_app_ui:
+
+User interface
+**************
+
+Most of the user interface mappings are common across all nRF5340 Audio applications.
+See the :ref:`nrf53_audio_app_ui` page for detailed overview.
+
+This application uses specific mapping for the following user interface elements:
+
+* Pressed on the unicast client device during playback:
+
+  * **PLAY/PAUSE** - Starts or pauses the playback of the stream.
+  * **VOL-** - Turns the playback volume down (and unmutes).
+  * **VOL+** - Turns the playback volume up (and unmutes).
+  * **BTN 4** - Sends a test tone generated on the device. Use this tone to check the synchronization of headsets.
+
+* **LED1** - Blinking blue on the CIS gateway: Kit has connected to a headset.
+* **RGB1** - Solid green - The device is programmed as the gateway.
```

Comment:
I don't think RGB1 is written anywhere one the board, so maybe change RGB1 to Center LED? That goes for all mentions of RBG1 in all files

Reply:
There is 1 occurrence of RGB1 and several of RGB in the hardware specs, so I'll keep it as **RGB**.